docs/sources/features/datasources/cloudwatch.md

```diff
@@ -103,14 +101,26 @@ Here is a minimal policy example:
 
 ### AWS credentials
 
-If Auth Provider is `Credentials file`, Grafana tries to get credentials in the following order.
+There are three different authentication methods available. `AWS SDK Default` performs no custom configuration at all and instead uses the [default provider](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html) as specified by the AWS SDK for Go. This requires you to configure your AWS credentials separately, such as if you've [configured the CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html), if you're [running on an EC2 instance](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/iam-roles-for-amazon-ec2.html), [in an ECS task](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-iam-roles.html) or for a [Service Account in a Kubernetes cluster](https://docs.aws.amazon.com/eks/latest/userguide/iam-roles-for-service-accounts.html).
+
+`Credentials file` corresponds directly to the [SharedCredentialsProvider](https://docs.aws.amazon.com/sdk-for-go/api/aws/credentials/#SharedCredentialsProvider) provider in the Go SDK. In short, it will read the AWS shared credentials file and find the given profile. While `AWS SDK Default` will also find the shared credentials file, this option allows you to specify which profile to use without messing with environment variables. It doesn't have any implicit fallbacks to other credential providers, and will fail if using credentials from the credentials file doesn't work.
```

Comment:
`Credentials file` corresponds directly to the [SharedCredentialsProvider](https://docs.aws.amazon.com/sdk-for-go/api/aws/credentials/#SharedCredentialsProvider) provider in the Go SDK. It will read the AWS shared credentials file and find the profile. While the `AWS SDK Default` option will also find the shared credentials file, this option allows you to specify which profile to use without disturbing with environment variables. It doesn’t have any implicit fallbacks to other credential providers, and will fail if using credentials from the credentials file doesn’t work.

Reply:
I don't think "disturbing environment variables" is a change for the better, it changes the meaning.

Reply:
Any other word other than "messing" is good. Feel free to change. Disturbing was a suggestion.

Reply:
Done.